git/gogit/go.mod

```diff
@@ -8,6 +8,7 @@ replace (
 	github.com/fluxcd/pkg/gitutil => ../../gitutil
 	github.com/fluxcd/pkg/ssh => ../../ssh
 	github.com/fluxcd/pkg/version => ../../version
+	github.com/go-git/go-git/v5 => github.com/fluxcd/go-git/v5 v5.0.0-20221019082644-b33dae782906
```

Comment:
why do we need a replace here?

Reply:
I removed it from the internal replace block and made a standalone replace instead.

In terms of why we need it here, we need to get the latest gogit here so we can accommodate the new API changes and have them tested at pkg level.